scripts/do/mithminite_scythe.zs

```diff
@@ -1079,3 +1079,8 @@ events.onProjectileImpactArrow(function (e as crafttweaker.event.ProjectileImpac
     }
   }
 });
+
+<entity:thaumadditions:mithminite_scythe>.onTick(function(entity) {
```

Comment:
Please test this feature.

Follow this steps:

1. In new creative falt world, set ViewDistance to 2
2. `/gamerule doMobSpawning false` + `/cofh killall *`
3. Took Scythe and shot ~20 projectiles to the North
4. Wait 11+ seconds
5. Use `/forge entity list` and then `/perf entities`

If /forge command show no scythes but /perf show there IS scythes - then your change does not helping.